Set-DomainUser.ps1

```diff
@@ -752,12 +896,173 @@ Function Activate-Platform {
+            updateAccountContent                   = $True
+            updateAccountProperties                = $False
+            initiateCPMAccountManagementOperations = $False
+            specifyNextAccountContent              = $False
+            renameAccounts                         = $False
+            deleteAccounts                         = $False
+            unlockAccounts                         = $False
+            manageSafe                             = $False
+            manageSafeMembers                      = $False
+            backupSafe                             = $False
+            viewAuditLog                           = $False
+            viewSafeMembers                        = $False
+            accessWithoutConfirmation              = $False
+            createFolders                          = $False
+            deleteFolders                          = $False
+            moveAccountsAndFolders                 = $False
+            requestsAuthorizationLevel1            = $False 
+            requestsAuthorizationLevel2            = $False
+        }
+        New-SafePermissions -pvwaAddress $pvwaAddress -pvwaToken $pvwaToken -safe $safe -safeMember "PVWAAppUser" -memberType "user" -safePermissions $PVWAAppUser
+        New-SafePermissions -pvwaAddress $pvwaAddress -pvwaToken $pvwaToken -safe $safe -safeMember "PVWAAppUsers" -safePermissions $PVWAAppUser
+        #PSMAppUsers
+        $PSMAppUsers = @{
+            useAccounts                            = $False 
+            retrieveAccounts                       = $True
+            listAccounts                           = $True
+            addAccounts                            = $False 
+            updateAccountContent                   = $False
+            updateAccountProperties                = $False
+            initiateCPMAccountManagementOperations = $False
+            specifyNextAccountContent              = $False
+            renameAccounts                         = $False
+            deleteAccounts                         = $False
+            unlockAccounts                         = $False
+            manageSafe                             = $False
+            manageSafeMembers                      = $False
+            backupSafe                             = $False
+            viewAuditLog                           = $False
+            viewSafeMembers                        = $False
+            accessWithoutConfirmation              = $False
+            createFolders                          = $False
+            deleteFolders                          = $False
+            moveAccountsAndFolders                 = $False
+            requestsAuthorizationLevel1            = $False 
+            requestsAuthorizationLevel2            = $False
+        }
+        Set-SafePermissions -pvwaAddress $pvwaAddress -pvwaToken $pvwaToken -safe $safe -safeMember "PSMAppUsers" -safePermissions $PSMAppUsers  
```

Comment:
Not sure why this is set and not new. I think it should also be new.

Reply:
I wonder if there's an environmental difference. In mine, PSMAppUsers was added to the safe automatically at creation time and trying to add it again resulted in an error, so I added the Set function to modify the existing user.